dist/api_data.py

```diff
@@ -575,15 +575,19 @@ def __cmp__(self, other):
         Config('file_max', '100MB', r'''
             the maximum size of log files''',
             min='100KB', max='2GB'),
-            Config('path', '"."', r'''
-                the name of a directory into which log files are written. The
-                directory must already exist. If the value is not an absolute
-                path, the path is relative to the database home (see @ref
-                absolute_path for more information)'''),
+        Config('path', '"."', r'''
+            the name of a directory into which log files are written. The
+            directory must already exist. If the value is not an absolute
+            path, the path is relative to the database home (see @ref
+            absolute_path for more information)'''),
         Config('recover', 'on', r'''
             run recovery or error if recovery needs to run after an
             unclean shutdown''',
-            choices=['error','on'])
+            choices=['error','on']),
+        Config('trickle', '100', r'''
```

Comment:
I prefer ``background_flush_period`` to ``trickle``. How did you choose 100ms as the default? I wonder if the default should be 0, since we don't really want to add in additional fsync calls for applications that configure synchronous logging.